### ISSUE TITLE
User dictionary list should be sorted

### ISSUE THREAD

**Issue by snomos** [divvun/giellakbd-ios#176]
Presently the list seems randomly sorted, cf screenshot:

![image](https://user-images.githubusercontent.com/1255285/95286864-2d55fe80-086d-11eb-9fc6-fe1bd4144571.jpeg)

- Not according to language name 
- Not according to ISO code
